lib/bibtex_ingester.rb

```diff
@@ -226,7 +226,7 @@ def convert_bibtex_record_to_pub_hash(record, author)
       identifiers << isbn_for_id_array
     end
     unless doi.blank?
-      doi_for_id_array = { type: 'doi', id: doi, url: 'http://dx.doi.org/' + doi }
+      doi_for_id_array = { type: 'doi', id: doi, url: "#{Settings.SCIENCEWIRE.DOI_BASE_URL}#{doi}" }
```

Comment:
If the `DOI` are a new settings group, this could become `Settings.DOI.BASE_URL`.